packages/transport/src/modules/relationships/RelationshipsController.ts

```diff
@@ -99,14 +99,20 @@ export class RelationshipsController extends TransportController {
     }
 
     public async getRelationshipToIdentity(address: CoreAddress, status?: RelationshipStatus): Promise<Relationship | undefined> {
-        const query: any = {
-            [`${nameof<Relationship>((r) => r.peer)}.${nameof<Identity>((r) => r.address)}`]: address.toString()
-        };
+        const simpleQuery: any = { peerAddress: address.toString() };
+        const dotNotationQuery: any = { [`${nameof<Relationship>((r) => r.peer)}.${nameof<Identity>((r) => r.address)}`]: address.toString() };
+
         if (status) {
-            query[`${nameof<Relationship>((r) => r.status)}`] = status;
+            simpleQuery[`${nameof<Relationship>((r) => r.status)}`] = status;
+            dotNotationQuery[`${nameof<Relationship>((r) => r.status)}`] = status;
```

Comment:
``${nameof<Relationship>((r) => r.status)}`` can be saved as variable to only call it once

Reply:
this is not performance-heavy and is more for making sure that we always query the correct props.